src/app/components/modules/gpa-calculator/gpa-calculator.component.ts

```diff
@@ -57,9 +57,11 @@ export class GpaCalculatorComponent extends SubscriptionsComponent
 		'F': 0.0
 	};
 
-	dropdownGradeInputs = ['N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A'];
+	dropdownGradeInputs: string[] = ['N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A', 'N/A'];
 	showOutput = false;
-	calculationOutputDisplayString = 'Something\'s Broken';
+	displayString  = 'Something\'s Broken';
```

Comment:
Can we make this either more informative to the user or just an empty string or something?